# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/mine/hotfix.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2003-09-29 00:26:40.000000000 +0200
+++ ./Makefile	2003-09-29 00:26:42.000000000 +0200
@@ -79,10 +79,10 @@
 all: $(targets)
 
 mine: $(MINE_ALL_OBJS)
-	gcc $(MINE_ALL_OBJS) $(MINE_LDFLAGS) -o mine
+	$(CC) $(MINE_ALL_OBJS) $(MINE_LDFLAGS) -o mine
 
 gasgui: $(GAS_OBJ)
-	gcc $(GAS_OBJ) -ldialog -lcurses -lm -o gasgui
+	$(CC) $(GAS_OBJ) -ldialog -lcurses -lm -o gasgui
 
 install:
 	install mine $(sysprefix)/bin/mine
--- ./selector.c	(revision 15)
+++ ./selector.c	(revision 16)
@@ -551,7 +551,7 @@
 		}
 
 		if ( !pkgcount && !bigmain ) {
-			char *dirname = "All Packages";
+			char *dirname = "all";
 			struct directory_entry * e;
 			if ( instmenu1() == 1 ) dirname = "base";
 			for (e = rootdir->list; e != NULL; e = e->next) {
